layers/+vim/evil-better-jumper/packages.el

```diff
@@ -30,10 +30,10 @@
     :init
     (global-set-key [remap evil-jump-forward]  #'better-jumper-jump-forward)
     (global-set-key [remap evil-jump-backward] #'better-jumper-jump-backward)
-    (global-set-key [remap xref-pop-marker-stack] #'better-jumper-jump-backward)
+    (global-set-key [remap xref-go-back] #'better-jumper-jump-backward)
     :config
     (better-jumper-mode 1)
-    (spacemacs|hide-lighter better-jumper-mode)
+    (global-set-key [remap xref-go-back] #'better-jumper-jump-backward)
```

Comment:
I assume this change is unintentional?  (If you could also please rebase your PR when you get the chance, to remove the extraneous bookmark/evil-collection change)

Reply:
No, it's not needed. Removed. Thank you!